v0/destinations/marketo/transform.js

```diff
@@ -547,7 +559,7 @@ const processEvent = async (message, destination, token) => {
   }
 
   // wrap response for router processing
-  return responseWrapper(response);
+  return responseWrapper(response, destination.ID);
```

Comment:
The `responseWrapper` function takes authKey as the second argument.

Reply:
Yes, here `destination.ID` is the `authKey`